tsconfig.common.json

```diff
@@ -20,6 +20,8 @@
     "declarationMap": true,
     "preserveConstEnums": true,
     "noImplicitAny": true,
+    // TODO: cleanup unknown usage in catch blocks and remove below setting
+    "useUnknownInCatchVariables": false
```

Comment:
the typescript update included no longer just letting untyped errors in catch blocks.  I started fixing these but there were a ton so I ended up just putting this place.

Reply:
So after cleaning up unknown usage, we change the value to true or remove the "useUnknownInCatchVariables" altogether?

Reply:
correct

Reply:
it defaults to true